core/templates/services/html-length.service.ts

```diff
@@ -16,6 +16,12 @@
  * @fileoverview Service for Computing length of HTML strings.
  */
 
+const CALCULATION_TYPE_WORD = 'word';
+const CALCULATION_TYPE_CHARACTER = 'character';
+const CUSTOM_TAG_REGEX = /<oppia-noninteractive-(?:math|image)[^>]*>/g;
+
+type CalculationType = 'word' | 'character';
```

Comment:
Can you make this refer to the constants above rather than hardcoding the strings? What do we do elsewhere in the codebase?

Reply:
I changed to constants, Mostly we use enums or making constants.